cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -927,6 +974,7 @@ def break_vm(self, vm_id):
         """ Method delete vm via virsh to emulate situation, when vm is valid
         and active in nova db, but in fact does not exist
         """
+        self.log.debug("Breaking VM with id %s.", vm_id)
```

Comment:
"Breaking VM..." is not clear, I'd suggest adding debug log of `virsh destroy && virsh undefine` command in addition to this message